node-commons/src/main/java/org/apache/iotdb/commons/udf/builtin/BuiltinAggregationFunction.java

```diff
@@ -56,4 +57,48 @@ public String getFunctionName() {
   public static Set<String> getNativeFunctionNames() {
     return NATIVE_FUNCTION_NAMES;
   }
+
+  /** @return if the Aggregation can use statistics to optimize */
+  public static boolean canUseStatistics(String name) {
+    final String functionName = name.toLowerCase();
+    switch (functionName) {
+      case "min_time":
+      case "max_time":
+      case "max_value":
+      case "min_value":
+      case "extreme":
+      case "first_value":
+      case "last_value":
+      case "count":
+      case "avg":
+      case "sum":
+        return true;
+      case "count_if":
+        return false;
+      default:
+        throw new IllegalArgumentException("Invalid Aggregation function: " + name);
+    }
+  }
+
+  /** @return if the Aggregation can split to multi phases */
+  public static boolean canSplitToMultiPhases(String name) {
```

Comment:
Strictly, not same. count_if can be used in group by level which means that it can `MergePartialResult`, but it can never be `SplitToMultiPhases` for each series